tests/tests/util/navigator.rs

```diff
@@ -125,4 +127,17 @@ impl NavigatorBackend for TestNavigatorBackend {
     fn pre_process_url(&self, url: Url) -> Url {
         url
     }
+
+    fn connect_socket(
+        &mut self,
+        _host: String,
+        _port: u16,
+        handle: SocketHandle,
+        _receiver: Receiver<Vec<u8>>,
+        sender: Sender<SocketAction>,
+    ) {
+        sender
```

Comment:
Since this is for the test backend, I think it might be preferable to log a socket connection here, similarly to how it  was done for `navigate_to_url` and `fetch`, so that testing might be a bit easier. For example:
```
if let Some(log) = &self.log {
    log.avm_trace("Navigator::connect_socket");
    log.avm_trace(&format!("    Host: {}; Port: {}", host, port));
}
```